doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+Support Status
+==============
+
+Levels of Support
+-----------------
+
+Tier 1
+~~~~~~
+
+Core team develops and supports. Compiler errors or functional
+failures block git merges and releases. Functionality is enabled by
+default on the platforms it supports.
+
+Tier 2
+~~~~~~
+
+Core team develops and supports, sometimes with help from community
+members. Compiler errors or functional failures block git merges.
+Functionality problems such as ‘known issues’ or might still going
+into releases. Functionality may be disabled by default.
+
+Community
+~~~~~~~~~
+
+When a feature of Suricata is community supported, it means the
+OISF/Suricata development team won’t support it. This to avoid
+overloading the team.
+
+When accepting a feature into the code base anyway, it will come with
+a number of limits and conditions:
+
+* submitter must commit to maintaining it:
+
+  - make sure code compiles and correctly functions after Suricata
+    and/or external (e.g. library) changes.
+  - support users when they encounter problems on the mailinglists and
+    redmine tickets
+
+* the code will be disabled by default and will not become part of the
+  QA setup. This means it will be enabled only by a —enable- configure
+  flag.
+
+If the feature get lots of traction, and/or if the team just considers
+it very useful, it may get ‘promoted’ to being officially supported.
+
+On the other hand, the feature will be removed if the submitter stops
+maintaining it and no-one steps up to take over.
+
+Unmaintained
+~~~~~~~~~~~~
+
+When a feature is unmaintained it is very likely broken and may be
+(partially) removed during cleanups and code refactoring. No end-user
+support is done by the core team. If someone wants to help maintain
+and support such a feature, we recommend talking to the core team
+before spending a lot of time on it.
```

Comment:
May be good to add a reference to: https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html#communication-is-key as we mention such things here and the channels for folks to reach out.